README.md

```diff
@@ -27,6 +27,7 @@ If you care and would like to make a contribution or join, you are welcome to do
     * [Pre-Release Disclosure Agreement](docs/pre-release-disclosure.md)
     * [Group charter](docs/charter.md) ⚠️  DRAFT
     * Standards and regulations [reading list](docs/readinglist.md) ⚠️  DRAFT
+    * [CVE Creation Guide](docs/cve-creation-guide.md)
```

Comment:
We have moved the files around since this was made; Better update the link to the new location? (`docs/guides/cve-creation-guide.md`)

Reply:
Merging the main branch into this one should work with a little tweaking.